_tools/release_docs/main.go

```diff
@@ -0,0 +1,158 @@
+package main
+
+import (
+	"errors"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"net/url"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"regexp"
+	"strings"
+)
+
+var (
+	// Finds markdown links of the form [foo](bar "alt-text").
+	linkRE = regexp.MustCompile(`\[([^]]*)\]\(([^)]*)\)`)
+	// Splits the link target into link target and alt-text.
+	altTextRE = regexp.MustCompile(`(.*)( ".*")`)
+
+	version = flag.String("version", "", "A version tag to process docs. (e.g. 1.0).")
```

Comment:
Specifically this determines the destination folder by prepending v to it, and determines the release tag by prepending release- to it.